db/migrate/20160901013042_change_column_type.rb

```diff
@@ -0,0 +1,14 @@
+class ChangeColumnType < ActiveRecord::Migration
+  def change
+    connection.execute(%q{
+        alter table shares
+        alter column lat
+        type float using lat::float
+    })
+    connection.execute(%q{
+        alter table shares
+        alter column lon
+        type float using lon::float
+    })
```

Comment:
This works and is fine, but you could also use the `change_column` migration command I believe.  Would be a little easier for future reference

http://edgeguides.rubyonrails.org/active_record_migrations.html